<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>1071</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>60</number>
      </property>
      <property name="midLineWidth">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Wallpaper">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>521</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>530</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>550</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>527</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="time">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>551</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Desktoploader">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>530</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Detector_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>991</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="FileUploads">
       <attribute name="title">
        <string>File Uploader</string>
       </attribute>
       <widget class="QPushButton" name="openFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Upload your file</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>321</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman','Times New Roman','Times New Roman'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;Welcome to the uploader to the plaigerism website !!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;This program was created to help doctors and students upload documents in order to check their plaigerism level &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;Press on &amp;quot;Upload file&amp;quot; to upload the file you are going to use&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;After Uploading the file press on which method you'd like to use&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="cursorWidth">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="BoyerMooreMatcher">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>60</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Boyer-Moore Algorithm</string>
        </property>
       </widget>
       <widget class="QPushButton" name="submitFile">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>350</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Submit Document</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>9</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Select The Method Of Detection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="KMPMatcher">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>130</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>KMP Algorithm</string>
        </property>
       </widget>
       <widget class="QPushButton" name="RabinKarpMatcher">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>200</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Rabin-Karp Algorithm</string>
        </property>
       </widget>
       <widget class="QPushButton" name="HammingDistanceMatcher">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>270</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Hamming Distance Algorithm</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Detector">
       <attribute name="title">
        <string>Plaigerism Detector</string>
       </attribute>
       <widget class="QTextEdit" name="colorindicater">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>521</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman','Times New Roman'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="percentage">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>320</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="MessageLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>69</y>
          <width>521</width>
          <height>41</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>991</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MinimizeButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <zorder>line</zorder>
     <zorder>Wallpaper</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>date</zorder>
     <zorder>time</zorder>
     <zorder>Desktoploader</zorder>
     <zorder>Detector_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>line_2</zorder>
     <zorder>tabWidget</zorder>
     <zorder>ExitButton</zorder>
     <zorder>MinimizeButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
